e2e/scenarios/open-allocations.ts

```diff
@@ -0,0 +1,49 @@
+import hre from 'hardhat'
+import { allocateFrom, stake } from './lib/staking'
+import { fundAccountsEth, fundAccountsGRT } from './lib/accounts'
+import { getIndexerFixtures } from './fixtures/indexers'
+import { fund } from './fixtures/funds'
+
+async function main() {
+  const graph = hre.graph()
+  const indexerFixtures = getIndexerFixtures(await graph.getTestAccounts())
+
+  const deployer = await graph.getDeployer()
+  const indexers = indexerFixtures.map((i) => i.signer.address)
+
+  // == Fund participants
+  console.log('\n== Fund indexers')
+  await fundAccountsEth(deployer, indexers, fund.ethAmount)
```

Comment:
Maybe I'd put `fund` amounts inside the Indexer fixture, like a `.grtBalance` and `.ethBalance` within each indexer, and maybe rename `fund` to `default` as placeholder?

Reply:
good idea